### ISSUE TITLE
V14-RC3: a11y: cannot tab to alias

### ISSUE THREAD

**Issue by rammi987** [umbraco/Umbraco-CMS#16246]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

14.0.0-rc3

### Bug summary

You can not tab to the lock and hit the spacebar to enter the alias texteditor to change the alias.
This is both on the doctype name and the property name, it simple jump over the lock as an action see video.

[screen-capture (3).webm](https://github.com/umbraco/Umbraco-CMS/assets/5937899/869e15fe-d305-42e1-be0d-d8f2decb591c)

### Steps to reproduce

Add a new doctype, enter the name and tab to the lock 
Add a new property, enter the name and tab to the lock

### Expected result / actual result

I expected after enter the name and tab the next highlighted element in the browser was the lock so i could detemind if i should press spacebar to tab into the editor and change the alias or should step over to the next action.

Same for property after enter the name and tab the next highligted element in the browser was the lock, so i could detemind if i should press spacebar to tab into the editor and change the alias or should step over to the next action witch are description

**Comment by Zeegaan** [comment]
I can reproduce this 👍 
Thanks for reporting, we will have a look 🐛
